### PR TITLE
Add LiDAR - Hesai AT128

### PR DESCRIPTION
**Pull request by msz-rai**
Added Hesai AT128 lidar.
PR is waiting for uploading released RGL binaries.

### CODE REVIEW

File: Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs [hunk 1]
```diff
@@ -273,6 +276,17 @@ public static float[] IntoMat3x4f(Matrix4x4 mat)
             return IntoMat3x4f(new[]{mat});
         }
 
+        public static float[] IntoVec2f(Vector2[] vecs)
```
Comment (PiotrMrozik) [inline Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs:279]: Are U sure name of this method is proper?

File: Assets/RGLUnityPlugin/Scripts/Utilities/DrawIfAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        }
+    
+        /// <summary>
+        /// Errors default to showing the property.
+        /// </summary>
+        private bool ShowMe(SerializedProperty property)
+        {
+            drawIf = attribute as DrawIfAttribute;
+            // Replace propertyname to the value from the parameter
+            string path = property.propertyPath.Contains(".") ? System.IO.Path.ChangeExtension(property.propertyPath, drawIf.comparedPropertyName) : drawIf.comparedPropertyName;
```
Comment (PiotrMrozik) [inline Assets/RGLUnityPlugin/Scripts/Utilities/DrawIfAttribute.cs:69]: fix indentation.

File: Assets/RGLUnityPlugin/Scripts/Utilities/DrawIfAttribute.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    
+            comparedField = property.serializedObject.FindProperty(path);
+    
+            if (comparedField == null)
+            {
+                Debug.LogError("Cannot find property with name: " + path);
+                return true;
+            }
+    
+            // get the value & compare based on types
```
Comment (PiotrMrozik) [inline Assets/RGLUnityPlugin/Scripts/Utilities/DrawIfAttribute.cs:79]:
```suggestion
            // Get the value & compare based on types
```

File: Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs (hunk 1 above)
Comment (msz-rai) [reply Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs:279]: Yes. It converts Vector2[] into Vec2f[] (RGL type that consists of two floats).

**Comment by msz-rai** [comment]
This PR will be converted to `Open` when released RGL binaries are updated.

**Pull request merged**